### ISSUE TITLE
Add information about using the recommended initial solution

### ISSUE THREAD

**Issue by JakubKudela89** [numbbo/coco-experiment#24]
The "recommended" initial solutions (all zeros for bbob, feasible solution for bbob-constrained,..) can be quite advantageous in some situations (particularly in higher dimensions). This information should appear somewhere in the description of how to run the experiments.

**Comment by brockho** [comment]
Agreed. To precise: we took care of this, for sure, in python but likely forgot to update the example experiments in other languages...
